src/EngineBuilder.cs

```diff
@@ -54,7 +73,45 @@ public class EngineBuilder
         /// <returns>Returns the current builder.</returns>
         public EngineBuilder WithDebugInfo(bool enable)
```

Comment:
I think this is basically serving the position of `Config`, right? If that's true perhaps for consistency with other bindings this could be called `Config`?

I was also curious as to why you stored each configuration field's value in this struct, and then have a "build" which actually feeds them all into a `wasm_config_t`. Alternatively the `EngineBuilder` could just hold on to the `wasm_config_t`, right?

Reply:
I think the original intention might have been to only throw from `Build` and not the various setter methods, but there's no reason for that really; it's certainly not worth the added complexity so I can remove the fields.

Regarding the name, I'm fine with renaming this to `Config` for consistency as now would be the time to do it.

Reply:
Actually, I believe the actual intention for why `EngineBuilder` was done this way was to solve the ownership transfer problem given that `wasm_engine_new_with_config`, sadly, takes ownership of the passed in config.

Having the config built entirely in the `Build` method prevented the `EngineBuilder` from having to implement `IDisposable`.